source/braille.py

```diff
@@ -872,7 +870,21 @@ def _addTextWithFields(self, info, formatConfig, isSelection=False):
 					if self._skipFieldsNotAtStartOfNode and not field.get("_startOfNode"):
 						text = None
 					else:
+						textList=[]
+						states=field.get('states')
+						if states and controlTypes.STATE_CLICKABLE in states:
+							if not inClickable:
+								# We have entered an outer most clickable or entered a new clickable after exiting a previous one 
+								# Report it if there is nothing else interesting about the field, but not if the user turned it off. 
+								if formatConfig['reportClickable']:

```

Comment:
Would it be possible to check this earlier on? I guess it is not necessary to check the clickable state if we don't report clickables at all.